tests/test_modeling_ctrl.py

```diff
@@ -219,6 +219,7 @@ class CTRLModelLanguageGenerationTest(unittest.TestCase):
     @slow
     def test_lm_generate_ctrl(self):
         model = CTRLLMHeadModel.from_pretrained("ctrl")
+        model = model.to(torch_device)
```

Comment:
models should just be model.to for consistency with other code

```suggestion
        model.to(torch_device)
```

Reply:
(and it saves a few chars)

Reply:
changed it everywhere :-)